pkgs/development/libraries/physics/rivet/default.nix

```diff
@@ -28,6 +28,8 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = [ fastjet ghostscript gsl yoda ];
 
   preConfigure = ''
+    substituteInPlace analyses/Makefile.in \
+      --replace "!(tmp)" ""
```

Comment:
What does this achieve?

Reply:
I don't know. This "!(tmp)" piece was inserted by upstream into the tarball release and it is not present in the official repository, it also breaks the build.